akka-http-tests/src/main/java/akka/http/javadsl/server/examples/simple/SimpleServerApp.java

```diff
@@ -4,84 +4,76 @@
 
 package akka.http.javadsl.server.examples.simple;
 
-import akka.actor.ActorSystem;
-import akka.http.javadsl.server.*;
-import akka.http.javadsl.server.values.Parameter;
-import akka.http.javadsl.server.values.Parameters;
-import akka.http.javadsl.server.values.PathMatcher;
-import akka.http.javadsl.server.values.PathMatchers;
+import static akka.http.javadsl.server.PathMatchers.IntegerSegment;
+import static akka.http.javadsl.server.PathMatchers.segment;
+import static akka.http.javadsl.server.StringUnmarshaller.INT;
+import static akka.http.javadsl.server.Unmarshaller.entityToString;
 
 import java.io.IOException;
-import java.util.concurrent.Callable;
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.CompletionStage;
+import java.util.concurrent.Executor;
+import java.util.function.BiFunction;
+import java.util.function.Function;
 
-public class SimpleServerApp extends HttpApp {
-    static Parameter<Integer> x = Parameters.intValue("x");
-    static Parameter<Integer> y = Parameters.intValue("y");
-
-    static PathMatcher<Integer> xSegment = PathMatchers.intValue();
-    static PathMatcher<Integer> ySegment = PathMatchers.intValue();
+import akka.actor.ActorSystem;
+import akka.http.javadsl.server.HttpApp;
+import akka.http.javadsl.server.Route;
 
-    static RequestVal<String> bodyAsName = RequestVals.entityAs(Unmarshallers.String());
+public class SimpleServerApp extends HttpApp {
 
-    public static RouteResult multiply(RequestContext ctx, int x, int y) {
+    public Route multiply(int x, int y) {
         int result = x * y;
-        return ctx.complete(String.format("%d * %d = %d", x, y, result));
+        return complete(String.format("%d * %d = %d", x, y, result));
     }
-    public static CompletionStage<RouteResult> multiplyAsync(final RequestContext ctx, final int x, final int y) {
-        return CompletableFuture.supplyAsync(() -> multiply(ctx, x, y), ctx.executionContext());
+    
+    public CompletionStage<Route> multiplyAsync(Executor ctx, int x, int y) {
+        return CompletableFuture.supplyAsync(() -> multiply(x, y), ctx);
     }
 
     @Override
     public Route createRoute() {
-        Handler addHandler = new Handler() {
-            @Override
-            public RouteResult apply(RequestContext ctx) {
-                int xVal = x.get(ctx);
-                int yVal = y.get(ctx);
-                int result = xVal + yVal;
-                return ctx.complete(String.format("%d + %d = %d", xVal, yVal, result));
-            }
+        Route addHandler = param(INT(), "x", x ->
```

Comment:
They're actually intended to become `public static final`s in a class. So then it'll just be `INT`. But I might just make this one `INTEGER`, since that's what the boxed class is called in Java.